back/spec/validators/json_validator_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe JsonValidator do
+  subject(:record) { Klass.new }
+
+  context 'with trivial string schema' do
+    before do
+      klass = Class.new do
+        include ActiveModel::Validations
+        validates :json_field, json: { schema: { type: 'string' } }
+        attr_accessor :json_field
+      end
+
+      stub_const('Klass', klass)
+    end
```

Comment:
Small detail, but do we really need to stub `Klass` here? Can't we just write:
```suggestion
  subject(:record) { klass.new }

  context 'with trivial string schema' do
    let(:klass) do 
      Class.new do
        include ActiveModel::Validations
        validates :json_field, json: { schema: { type: 'string' } }
        attr_accessor :json_field
      end
    end
```

Reply:
Yes We Can! :+1: